### PR TITLE
Empty address

### PR DESCRIPTION
**Pull request by aswen**

### CODE REVIEW

File: templates/interface/Debian.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@ allow-hotplug <%= @interface %>
 <% if @manage_address -%>
     address <%= @manage_address %>
 <% end -%>
-<% if @manage_hwaddr -%>
+<% if @manage_address and ! @manage_address.empty? -%>
```
Comment (alvagante) [inline templates/interface/Debian.erb]: Why adding check on manage_address here?
It would not not work with dhcp interfaces.

Comment (aswen) [inline templates/interface/Debian.erb]: With dhcp interfaces that line should not be there. And now it's set to '' (empty string) which evaluates to true resulting in lines with just the word address in stanza's that don't need that line.

Comment (alvagante) [inline templates/interface/Debian.erb]: Sorry, I don't follow you.
Why on dhcp interfaces the  hwaddr entry should not be there?
Are you sure that the line:
<% if @manage_address and ! @manage_address.empty? -%>
should be before hwaddr and not address?

Comment (aswen) [inline templates/interface/Debian.erb]: ohh. crap... I updated this pr the wrong way. Xcuse me! line 12 should be at line 9. I'll update my pr strait away.

File: templates/interface/Debian.erb [hunk 2]
```diff
@@ -6,10 +6,10 @@ auto <%= @interface %>
 allow-hotplug <%= @interface %>
 <% end -%>
 <%= @stanza %> <%= @interface %> <%= @family %> <%= @manage_method %>
-<% if @manage_address -%>
+<% if @manage_address and ! @manage_address.empty? -%>
     address <%= @manage_address %>
 <% end -%>
-<% if @manage_hwaddr -%>
+<% if @manage_address -%>
```
Comment (alvagante) [inline templates/interface/Debian.erb]: Here should be: 
<% if @manage_hwaddr -%>

Comment (aswen) [inline templates/interface/Debian.erb]: how could i miss that?

**Comment by alvagante** [comment]
thanx

**Pull request merged**